### DISCUSSION TITLE
UGreen CM448 RTL8811CU Working

### DISCUSSION THREAD

**Discussion by vxdust** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#321] [Ideas]
macOS Monterey 12.7.2 Hackintosh
![Screen Shot 2024-01-13 at 21 00 17](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/147116635/c284d9e6-28e5-4041-98bd-8c94c82d1eb9)

![9d3af651ab1e48437da1f20da2ab5c71](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/147116635/39497255-b441-4242-81d9-890d95d6ca36)

**Comment by chris1111** [comment]
Thanks [ Done](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/0cacd7021f7c9d6d6188230eec6c21c8dfc40514)
